### DISCUSSION TITLE
largeblob webauthn / winhello

### DISCUSSION THREAD

**Discussion by voibit** [Yubico/libfido2#838] [Q&A] [answered]
Hi,
I have an offline use case for a yubikey. However, it seems the credman extension isn't working with Windows Hello (it works with elevated privileges). 

Does libfido2 support any other ways to access large blobs, large blob keys,  or resident key public keys through WebAuthn on Windows?

**Comment by LDVG** [comment, marked as answer]
Hi,

We have no support for credential management or large blobs using the `winhello` backend, sorry.
